src/deprecated.jl

```diff
@@ -357,3 +357,32 @@ function by(d::AbstractDataFrame, cols::Any; sort::Bool=false, skipmissing::Bool
 end
 
 @deprecate groupvars(gd::GroupedDataFrame) groupcols(gd)
+
+@deprecate aggregate(d::AbstractDataFrame, fs::Any; sort::Bool=false) begin
+    df = select(d, names(d) .=> [fs])
```

Comment:
This seems to be enough:
```suggestion
    df = select(d, names(d) .=> fs)
```

Reply:
It would not be equivalent to what `aggregate` did in the past (but maybe it is not a problem - the differences will be apparent only in very obscure cases when some custom broadcasting is defined for `fs`, or it s). So - given this - what do you think - leave what I have written (which is safe) or what you propose (which is cleaner but sometimes might fail).

Reply:
Actually I rewrite it anyway. So I resolve it.